<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileWindow</class>
 <widget class="QDialog" name="ProfileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Профиль</string>
  </property>
  <property name="windowIcon">
   <iconset resource="IMG.qrc">
    <normaloff>:/IMG/IMG/logoSN.png</normaloff>:/IMG/IMG/logoSN.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 249, 255);</string>
  </property>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="phoneTextLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
    background-color: rgb(255, 0, 4);
    color: white;     
    border-radius: 20px;
    padding: 8px 16px;  
    font-weight: bold;   
    border: 1px solid #4a148c; 
}

QPushButton:hover
 {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #7b1fa2,
        stop:1 #aa00ff
    );
    border: 2px solid #7b1fa2;
}

QPushButton:pressed 
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #4a148c,
        stop:1 #7b1fa2
    );
}</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="phoneLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: grey;</string>
   </property>
   <property name="text">
    <string>Телефон:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: grey;</string>
   </property>
   <property name="text">
    <string>Дата рождения:</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>361</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="loginUserLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: grey;</string>
   </property>
   <property name="text">
    <string>Логин:</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>361</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>15</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
	border-radius: 17px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IMG.qrc">
     <normaloff>:/IMG/IMG/SettingsPin35x35SN.png</normaloff>:/IMG/IMG/SettingsPin35x35SN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="emailTextLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteQueryButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
    background-color: rgb(173, 173, 173);
    color: white;     
    border-radius: 20px;
    padding: 8px 16px;  
    font-weight: bold;   
    border: 1px solid #4a148c; 
}

QPushButton:hover
 {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #7b1fa2,
        stop:1 #aa00ff
    );
    border: 2px solid #7b1fa2;
}

QPushButton:pressed 
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #4a148c,
        stop:1 #7b1fa2
    );
}</string>
   </property>
   <property name="text">
    <string>Отменить заявку</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginTextLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dateTextLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="profileButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
	background-color: white;
	border: 1px solid;
    border-radius: 75px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IMG.qrc">
     <normaloff>:/IMG/IMG/DefultProfile150x150SN.png</normaloff>:/IMG/IMG/DefultProfile150x150SN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: grey;</string>
   </property>
   <property name="text">
    <string>Почта:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changePhotoButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
	background: transparent;
	border: 1px solid;
    border-radius: 75px;
	opacity: 0.5;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IMG.qrc">
     <normaloff>:/IMG/IMG/CameraPin150x150SN.png</normaloff>:/IMG/IMG/CameraPin150x150SN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
    background-color: rgb(38, 255, 0);
    color: white;     
    border-radius: 20px;
    padding: 8px 16px;  
    font-weight: bold;   
    border: 1px solid #4a148c; 
}

QPushButton:hover
 {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #7b1fa2,
        stop:1 #aa00ff
    );
    border: 2px solid #7b1fa2;
}

QPushButton:pressed 
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #4a148c,
        stop:1 #7b1fa2
    );
}</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="answerQueryButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6a3093, 
        stop:1 #a044ff 
    );
    color: white;     
    border-radius: 20px;
    padding: 8px 16px;  
    font-weight: bold;   
    border: 1px solid #4a148c; 
}

QPushButton:hover
 {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #7b1fa2,
        stop:1 #aa00ff
    );
    border: 2px solid #7b1fa2;
}

QPushButton:pressed 
{
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #4a148c,
        stop:1 #7b1fa2
    );
}</string>
   </property>
   <property name="text">
    <string>Ответить на заявку</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="IMG.qrc"/>
 </resources>
 <connections/>
</ui>
